### DISCUSSION TITLE
Coding Conventions

### DISCUSSION THREAD

**Discussion by DotNetDublin** [BlazingSaddlesSolutions/TimeKeeper#34] [General]
Not an immediate concern however we may look to subscribe to a set of coding conventions.

As a start does anyone have any issues with those described in the below link?

https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/inside-a-program/coding-conventions

I didn't find Microsoft recommendations for SQL databases and while I found the below we can probably simply put together a table or two, peer review it and then ask everyone to follow that format.

https://github.com/ktaranov/sqlserver-kit/blob/master/SQL%20Server%20Name%20Convention%20and%20T-SQL%20Programming%20Style.md

**Comment by harperjohn** [comment]
I am open to any coding convention as long as it can be supported/enforced by the IDE (or plugin)  :)   (My old and tired brain tends to "flashback" - somewhat randomly - to one of the many/various coding conventions that I have used during the last 35+ years :) )
